src/renderer/components/OurTextArea/OurTextArea.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import TextField from '@mui/material/TextField';
+
+interface Props {
+  type?: string;
+  error?: boolean;
+  onChange: (...args: any[]) => void;
+  value?: string;
+  helperText?: string;
+  placeholder?: string;
+  width?: string;
+}
+
+const StyledTextArea = styled(TextField)(({ type }) => ({
+  '& .MuiOutlinedInput-root': {
+    '& fieldset': {
+      borderRadius: '0px',
+      borderColor: type === 'light' ? lightGray.main : black.main,
+      color: type === 'light' ? lightGray.main : black.main,
+    },
+    '&.Mui-error': {
+      '& fieldset': {
+        borderRadius: '0px',
+        borderColor: lightRed.main,
+      },
```

Comment:
You're using `ligthGray` and `black` and `lightRead` but they do not come from anywhere. They are not defined at all.